engine/emoji-picker

```diff
@@ -16,8 +16,8 @@
 #  You should have received a copy of the GNU General Public License
```

Comment:
@mike-fabian Quick update, the designer agreed to work on this. I suggest to wait until the icons are ready :+1: